index.html

```diff
@@ -0,0 +1,125 @@
+    <title>Traffic & Conversion Summit</title>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'> <!-- Import Google Font -->
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link rel="stylesheet" href="./src/css/styles.css">
+  </head>
+
+  <body>
+    <header>
+      <!--  Begin HEADER     -->
+      <div class="topBar">
+            <ul class="social_icons">
+                <li><a class="socialimg" href="https://www.facebook.com"><img src="./src/images/facebook.png" alt="Facebook icon"></a></li>
+                <li><a class="socialimg" href="https://www.twitter.com"><img src="./src/images/twitter.png" alt="Twitter icon"></a></li>
+                <li>English</li>
+            </ul>
+        </div>
+      <nav class="navbar">
+        <div class="mobileNavbar">
+          <div class="navbarItem1">
+               <a class="hamburguer" href="#"><img src="./src/images/hamburguerIcon.png" alt="hamburguer"></a>
+          </div>
+           <div class="navbarItem2">
+               <a class="logo" href="#"> <img src= "./src/images/logoBlack.jpg" alt"logo"></a>
+          </div>
+          <div class="mobileMenu">
+            <ul class="itemsMobileMenu">
+              <li class="close_menu"> <img src="./src/images/close_menu.png" alt="Close"/></li>
+              <li class="itemProgram"> <a href="#mainProgram">Program</a> </li>
+              <li class="itemSpeaker"> <a href="#speaker">Speaker</a> </li>
+              <li class="itemAbout"> <a href="about.html">About</a> </li>
+              <li class="itemAbout"> <a href="index.html">Home</a> </li>
+            </ul>
+            <ul class="itemsDesktopMenu">
+              <li class="itemProgram"> <a href="#mainProgram">Program</a> </li>
+              <li class="itemSpeaker"> <a href="#speaker">Speaker</a> </li>
+              <li class="itemAbout"> <a href="about.html">About</a> </li>
+              <li class="itemAbout"> <a href="index.html">Home</a> </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+
+    <main>
+        <div class="hero">
+          <div class="card1">
+          <h1 class="home_title">TRAFFIC & CONVERSION SUMMIT 2022 </h1>
+          <h2 class="home_title2">The congress of online advertising!</h2>
+          <p class="description"> More than 10 hours of training divided into three days in which you will learn from active professionals knowledge, tips, and strategies of online advertising and digital strategy. Whether you're starting from scratch or already have experience,.</p>
+          <div class="dateInfo"> <font>2022.09.13 (Tue) ~ 15 (Thu)</font> </div>
+          <div class="locate">
+            <font>@ Sports Center San Miguel de Sema Boyaca - Colombia</font>
+          </div>
+        </div>
+      </div>
```

Comment:
- [ ] Please fix this section to match the design. This section is overflowing in both desktop and mobile versions, kindly fix this, please.
![1](https://user-images.githubusercontent.com/94551484/190757102-095a3218-019f-4cca-99ee-f726e89ab741.png)
![11](https://user-images.githubusercontent.com/94551484/190757333-8cb0c9ff-cd43-4457-9685-28ca025e6b41.png)